dpreference/src/test/java/ledongli/dpreference/ExampleUnitTest.java

```diff
@@ -9,7 +9,7 @@
  */
 public class ExampleUnitTest {
     @Test
-    public void addition_isCorrect() throws Exception {
+    public void additionisCorrect() {
```

Comment:
Test case not relevant to the library. Please delete the file.

Reply:
Done